src/pages/team/[team].astro

```diff
@@ -0,0 +1,71 @@
+---
+import Layout from "../../layouts/Layout.astro";
+//import AboutSection from "../components/AboutSection.astro";
+//import Hero from "../components/Hero.astro";
+import TeamCard from "../../components/TeamCard.astro";
+import TeamTab from "../../components/TeamTab.astro";
+
+import ADVISORS from "../../data/advisors.json";
+import CTMS from "../../data/ctms.json";
+import EXECUTIVES from "../../data/execs.json";
+
+export function getStaticPaths() {
+  return [
+    { params: { team: "Core Team Members" } },
+    { params: { team: "Executives" } },
+    { params: { team: "Advisors" } },
+  ];
+}
+
+const team = Astro.params.team as string;
+
+var team_members = ADVISORS;
```

Comment:
```suggestion
let team_members = ADVISORS;
```

`var` is deprecated. See [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) for more information.